Formula/acme.rb

```diff
@@ -1,10 +1,9 @@
 class Acme < Formula
   desc "Crossassembler for multiple environments"
-  homepage "https://web.archive.org/web/20150520143433/https://www.esw-heim.tu-clausthal.de/~marco/smorbrod/acme/"
-  url "https://www.mirrorservice.org/sites/ftp.cs.vu.nl/pub/minix/distfiles/backup/acme091src.tar.gz"
-  mirror "http://ftp.lip6.fr/pub/minix/distfiles/backup/acme091src.tar.gz"
-  version "0.91"
-  sha256 "31ed7f9be5cd27100b13d6c3e2faec35d15285542cbe168ec5e1b5236125decb"
+  homepage "https://sourceforge.net/projects/acme-crossass/"
+  url "https://downloads.sourceforge.net/code-snapshots/svn/a/ac/acme-crossass/code-0/acme-crossass-code-0-97-trunk.zip"
```

Comment:
Please ask upstream to provide stable archives for releases.

Reply:
asked, and they don't do it.

Reply:
Let's use `url "https://svn.code.sf.net/p/acme-crossass/code-0/trunk", :revision => "97"` instead of the sourceforge zip (and remove the `sha256` below).